### PR TITLE
Run build in spec

### PR DESCRIPTION
**Pull request by hugobho**
@kevgo @trushton

### CODE REVIEW

File: exoservice-javascript/bin/spec [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
+webpack
```
Comment (kevgo) [inline exoservice-javascript/bin/spec]: Let's follow our convertion for helper scripts:
- `build` compiles source code into executable formats, for example via `lsc` or `webpack`
- `spec` calls `build`

**Comment by hugobho** [comment]
Ready for another review. I've also made changes to the other subprojects to run `build` in `spec`

File: exocom-mock-javascript/bin/spec [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-node_modules/o-tools-livescript/bin/build
+build
```
Comment (kevgo) [inline exocom-mock-javascript/bin/spec]: I think this should be `bin/build`, since it will be called from the `exocom-mock-javascript` directory.

**Comment by hugobho** [comment]
@kevgo comments addressed.

**Pull request closed without merging**